cookbooks/bach_repository/metadata.rb

```diff
@@ -12,3 +12,4 @@
 depends 'ark'
 depends 'bcpc'
 depends 'build-essential'
+depends 'java'
```

Comment:
surprising there's no merge conflict here since @leochen4891 's jvmkill patches were merged in.